processors/data_contributions/processor.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import requests
+import logging
+from .. import base
+from . import extractors as extractors_module
+logger = logging.getLogger(__name__)
+
+
+def process(conf, conn):
+    def convert_ids(contrib):
+        converted_ids = {key: val.hex for key, val in contrib.items()
+                        if val.__class__.__name__ == 'UUID'}
```

Comment:
In general, instead of checking for the class itself, it's better to check for the method you want. In this case, you could do `hasattr(val, 'hex')`.